rsocket-core/src/commonMain/kotlin/io/rsocket/RSocket.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.rsocket
```

Comment:
You are going to need a different package, this one's taken :)

Reply:
We can use or just `rsocket` (why not?) or as now `io.rsocket.kotlin`
Not sure about that...

Reply:
@yschimke do you have any issues with `io.rsocket`? Are you mixing rsocket-java / kotlin? We discussed that with @whyoleg and I proposed initially to go with `io.rsocket` unless someone complains. And here this happened. Thus, the question is how much it is critical. If it is critical I would go for `kotlin.rsocket` or `io.rsocket.kotlin`

Reply:
It's a bad idea. Noone does this. How about io.rsocket.kt?

I can't suggest this strongly enough. Don't reuse the package.

Reply:
do you think `kotlin.rsocket` will confuse someone?

Reply:
anyways. I'm good with `io.rsocket.kt` just wanna have less `.` in the base pkg name

Reply:
I'll defer to you both, but just dont overlap.

Reply:
sure

Reply:
Let's then use `io.rsocket.kotlin`, I don't really like `kt` ending. If it's ok, I will change it soon

Reply:
Sounds good to me